projects/iio_demo/Makefile

```diff
@@ -1,7 +1,23 @@
 TARGET := iio_demo
 TINYIIOD ?= y
+ifeq (aducm3029,$(strip $(PLATFORM)))
+
+SRC_DIRS += projects/$(TARGET)/src
+SRC_DIRS += iio/iio_app
+SRC_DIRS += iio/iio_demo
+
+ifeq (y,$(strip $(TINYIIOD)))
+SRC_DIRS += libraries/iio
+endif
+
+include ../../tools/scripts/aducm.mk
```

Comment:
It seems to me like this change should not happen here. The existing makefiles compile for every platform with respect to the operating system in which the compilation is done. The new makefile aducm.mk compiles for the aducm platform with respect to the operating system. Should support for the aducm platform be added to the windows and linux makefiles instead?

Reply:
That way, this file does not change anymore, I believe.

Reply:
I think we should create a generic makefile for both windows and linux that do what this makefile dose or create a generic makefile slited in multiple file with specific functionalities.
But I think this is not related to this PR